### DISCUSSION TITLE
the ICA result do not show clearly

### DISCUSSION THREAD

**Discussion by wenboyu0803** [craddm/eegUtils#145] [Q&A]
Hi, 
i have already some analysis on eeg data.
After running ica, i found that topographies could not be fully displayed, but when i see individual componet, each of them could be displayed.
is there any method to solve this issue?
thanks a lot.
![ica](https://github.com/user-attachments/assets/480c2784-2e92-4e4f-b821-f599ff39c17f)

**Comment by craddm** [comment]
This is to do with the scale chosen for this plot. At the moment it's not possible to change the scale for the visualization here. When you look at them individually, the scale will match the range of the component. But for this plot, the scale is just set to +-3 standard deviations (IIRC). And the activity on most components is much much smaller than the activity on the first couple of components. 

Incidentally, this looks like quite a poor quality decomposition. Without being able to see the raw data it's not possible for me to be sure why. But I at least suspect that the electrode at the centre of the blue spot on the first two components (and subsequently in the red spot on many other components) - a left frontal electrode - is bad and might need to be excluded/interpolated before running the ICA.

**Comment by wenboyu0803** [comment]
Yes, you are right.

My expriment is a auditory task and My data is a little poor. This is the butterfly plot after i remove three bad channels.
![QQ20241003-105621](https://github.com/user-attachments/assets/4ee23974-2872-4464-9943-b0a97aa56bbc)

I also found that i could not delete epochs by funcition of ar_thresh, so my average amplitudes is quite larger, maybe over 500 uV.
Another issue i do not know how to handle is participants take rest during recoding, i should cut it off?

If possible, would you mind to see my raw.data?

Thanks you very much.

Wenbo Yu

**Reply by craddm** [reply]
It looks like you are using continuous data. There are no epochs to delete - it's all just one big epoch. `ar_thresh` will just delete individual timepoints. 

The values you are seeing there are enormous - way out of the range I'd expect. Not sure what system you are recording with. Have you referenced the data since importing it?

**Comment by wenboyu0803** [comment]
we used BP to record eeg data with Cz as online reference.
the data showed is not referenced, but even after i referenced, the sd of data is still quite.
[data.xlsx](https://github.com/user-attachments/files/17724185/data.xlsx)
